common/scripts/get-agentproxy

```diff
@@ -121,7 +121,7 @@ checkDesiredVersion() {
 # if it needs to be changed.
 checkAgentGatewayInstalledVersion() {
   if [[ -f "${AGENTGATEWAY_INSTALL_DIR}/${BINARY_NAME}" ]]; then
-    local version=$("${AGENTGATEWAY_INSTALL_DIR}/${BINARY_NAME}" version --template="{{ .Version }}")
+    local version=$("${AGENTGATEWAY_INSTALL_DIR}/${BINARY_NAME}" --version)
```

Comment:
This doesn't seem like it would work since `--version`

returns
```
agentgateway-app version.BuildInfo{RustVersion:"1.88.0", BuildProfile:"release", BuildStatus:"Modified", GitTag:"v0.6.0", Version:"2c7ba0d4ed47fcafa97fa411fdbf1a8ca40cf6a9-dirty", GitRevision:"2c7ba0d4ed47fcafa97fa411fdbf1a8ca40cf6a9-dirty"
```